src/wasm-ir-builder.h

```diff
@@ -55,9 +55,15 @@ class IRBuilder : public UnifiedExpressionVisitor<IRBuilder, Result<>> {
   // initialized to initialize the child fields and refinalize it.
   Result<> visit(Expression*);
 
+  // Whether an expression is organic (comes from the wasm) or synthetic (is
+  // generated automatically, e.g., a local.get for a scratch local). Only
+  // organic expressions have binary locations and debug info.
+  enum Origin { Organic = 0, Synthetic = 1 };
```

Comment:
Maybe `Real` instead of `Organic`?

Reply:
Another option is just Binary actually? `Origin=Binary` seems the least ambiguous of all options so far.